go-controller/pkg/ovn/controller/apbroute/external_controller_policy.go

```diff
@@ -271,12 +294,12 @@ func (m *externalPolicyManager) applyDiscrepanciesInRoutePolicy(currentPolicy, n
 }
 
 func (m *externalPolicyManager) removeDiscrepanciesInRoutePolicy(currentPolicy, updatedPolicy *adminpolicybasedrouteapi.AdminPolicyBasedExternalRoute) error {
-	unmachingNamespaces, unmatchingStaticHops, unmatchingDynamicHops, err := m.calculatePolicyDifferences(currentPolicy, updatedPolicy)
+	unmaching, unmatchingStaticHops, unmatchingDynamicHops, err := m.calculatePolicyDifferences(currentPolicy, updatedPolicy)
```

Comment:
unmatching*

Reply:
why remove Namespaces word? i think that added some clarity as to what the variable was